internal/commands/report.go

```diff
@@ -17,7 +17,7 @@ import (
 	"github.com/buildpacks/pack/logging"
 )
 
-func Report(logger logging.Logger) *cobra.Command {
+func Report(logger logging.Logger, client PackClient) *cobra.Command {
```

Comment:
This seems to go unused. What's the purpose of passing it in?

Reply:
Removed.